css/properties/position-try-fallbacks.json

```diff
@@ -1,16 +1,23 @@
 {
   "css": {
     "properties": {
-      "position-try-options": {
+      "position-try-fallbacks": {
         "__compat": {
-          "spec_url": "https://drafts.csswg.org/css-anchor-position-1/#position-try-options",
+          "spec_url": "https://drafts.csswg.org/css-anchor-position-1/#position-try-fallbacks",
           "tags": [
             "web-features:anchor-positioning"
           ],
           "support": {
-            "chrome": {
-              "version_added": "125"
-            },
+            "chrome": [
+              {
+                "version_added": "128"
+              },
+              {
+                "alternative_name": "position-try-options",
+                "version_added": "125",
+                "version_removed": "127"
```

Comment:
```suggestion
                "version_removed": "128"
```

Reply:
Yes! Sorry, I always get confused about which version I need to put in `"version_removed"`.